support-frontend/app/controllers/Application.scala

```diff
@@ -116,8 +115,6 @@ class Application(
   }
 
   def unsupportedBrowser: Action[AnyContent] = NoCacheAction() { implicit request =>
-    BrowserCheck.logUserAgent(request)
```

Comment:
Why is this no longer needed? Is it too spammy?

Reply:
oh yeah I forgot to mention in the description it's really frustrating to scroll past them all in the logs sO I got around to removing it.

Same goes for the `SafeLogger.info(s"unable to authorize user - $error")` below that I also removed.